modules/openfast-registry/src/main.cpp

```diff
@@ -13,6 +13,7 @@ Usage: openfast_registry registryfile [options] -or-
     -h                this summary
     -I <dir>          look for usefrom files in directory "dir"
     -O <dir>          generate types files in directory "dir"
+    -inc              generate types file to be included in another file
```

Comment:
Should this be `subs` for the new flag?

Reply:
changed to incsubs and updated documentation in new commit